### PR TITLE
test(e2e,playwright): Landing, Header

### PR DESCRIPTION
**Pull request by onukwilip**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705 - Landing

### CODE REVIEW

**Comment by onukwilip** [comment]
camperbot cant code

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
This will need a rebase with `main`. If you have questions on how to do that, feel free to pop in our Discord and we'll be happy to help.

**Comment by onukwilip** [comment]
Thanks ☺️! Will do that

**Comment by onukwilip** [comment]
Thanks for the review! I've successfully rebased with the `main` branch as per the steps you provided on the Discord server 😊

File: e2e/landing.spec.ts [hunk 1]
```diff
@@ -65,6 +65,31 @@ test('Has 5 brand logos', async () => {
...
+  }
+});
+
+test('The landing-top & testimonial sections call to action should contain a descriptive text content', async () => {
+  const ctas = page.locator("[data-test-label='landing-big-cta']");
+
+  for (const cta of await ctas.all()) {
+    await expect(cta).toHaveText(translations.buttons['logged-in-cta-btn']);
+  }
+});
```
[the remark this answers is not in the document]
Comment (onukwilip) [reply e2e/landing.spec.ts:82]: Ok, thanks. Will resolve that ☺️

**Comment by onukwilip** [comment]
Thanks for the help ☺️. I'm apply the requested changes

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍 Thank you for your contribution @onukwilip!

**Comment by onukwilip** [comment]
Happy to ☺️

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**